### PR TITLE
Append an exception for execution permission of gradlew on format

### PR DESCRIPTION
**Pull request by YongseopKim**
Append an exception for execution permission of gradlew on format
- FYI. gradlew should be execute-able for gradle

Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
PTAL @hseok-oh @chunseoklee @periannath

File: infra/command/format [hunk 1]
```diff
@@ -99,7 +99,7 @@ function check_permission() {
   FILES_TO_CHECK_PERMISSION=()
   for f in ${FILES_TO_CHECK[@]}; do
     # Manually ignore permission checking
-    if [[ ${f} == !(nnas|nnfw|nncc|*.sh|*.py) ]]; then
+    if [[ ${f} == !(nnas|nnfw|nncc|*.sh|*.py|gradlew) ]]; then
```
Comment (hseok-oh) [inline infra/command/format:102]: `|gradlew` -> `|*/gradlew`

Comment (YongseopKim) [reply infra/command/format]: Thanks!

**Comment by YongseopKim** [comment]
ping @Samsung/nnfw_committers

**Pull request merged**